# Message to request different follow behavior

Header header

bool Ball
bool Goal

uint8 BallCenterX
uint8 BallCenterY 
uint8 BallRadius

uint8 GoalX
uint8 GoalY
uint8 GoalW
uint8 GoalH

float32 DistBall
float32 DistGoal
